woocommerce/woocommerce-ios
# Simple Payments: Show payment link row

# Why

This PR adds the correspondent UI & flow control to show the payment link row when:
1. A valid payment link can be built
2. The feature flag is enabled

I have added a feature flag to wait for final confirmation while the relevant parties from products and design are AFK.

# Demo

https://user-images.githubusercontent.com/562080/147483260-5569d8f1-bbbc-4360-88a2-dcda936912a7.mov

# Testing Steps
- Start the Simple Payments flow
- Go all the way until the payment methods screen
- See the payment link row available
- Choose a share action
- See the flow is dismissed and that a notice is presented.

---
